low-level/sensors/impl/generic/node_hw.py

```diff
@@ -188,8 +171,9 @@ def __init__(self):
         except (IOError, ConfigReader.Error) as err:
             logger.error("[ Error ] when validating the config file {0} - {1}"\
                  .format(self.CONF_FILE, err))
-        self.polling_interval = int(Conf.get(SSPL_CONF, f"{self.NODEHWSENSOR}>{self.POLLING_INTERVAL}",
-                            self.DEFAULT_POLLING_INTERVAL))
+        self.polling_interval = int(Conf.get(SSPL_CONF,
+                    f"{NODEHWSENSOR}>{self.POLLING_INTERVAL}",
```

Comment:
continuation line under-indented for visual indent.

Reply:
addressed.